scene/gui/text_edit.h

```diff
@@ -816,6 +819,7 @@ class TextEdit : public Control {
 	void remove_secondary_carets();
 	void merge_overlapping_carets();
 	int get_caret_count() const;
+	void insert_caret_at_carets(bool p_below);
```

Comment:
If we really want to make this public, we should bind it to the gdscript API as well.

With that said, I think using `add` rather then `insert` sound better here. Not only to align with `add_caret` but also keep `insert` related to text operations.

Would it be worth this making one private and creating `add_caret_below_carets()` and `add_caret_above_carets()` as clearer names?

Reply:
Since most users will probably not even know that the method exists, I think it is enough if the documentation specifies that it's about adding a caret above or below and what the parameter does.